src/compounds/LinkedButton/LinkedButton.jsx

```diff
@@ -31,10 +31,10 @@ const { onClick, ...remainingPropTypes } = Button.propTypes
 LinkedButton.propTypes = {
   buttonProps: PropTypes.shape(remainingPropTypes).isRequired,
   addClass: PropTypes.string,
-  path: PropTypes.exact({
+  path: PropTypes.oneOfType([PropTypes.shape({
     pathname: PropTypes.string.isRequired,
     query: PropTypes.shape({ id: PropTypes.number }),
-  }).isRequired,
+  }), PropTypes.string]).isRequired,
```

Comment:
instead of allowing options here, can we update all instances of path to use the `{ pathname, query  }` format? this this code can remain as is and the rest of our code will be more uniform.

that can be pulled into a different pr though.

Reply:
i tried to do this and there is just one place where it is an issue. In the webstore, on the initiate request button for the make a request ware: `pages/requests/index.js`

this is how it started
<img width="584" alt="image" src="https://user-images.githubusercontent.com/73361970/215583762-29c494d3-c44f-4dca-9485-b6677fb4c466.png">

i tried this and several other things. not sure if it will work since this is really a hardcoded link vs a dynamic query 
<img width="740" alt="image" src="https://user-images.githubusercontent.com/73361970/215583363-bcae2f1e-8607-40e4-842a-834793b94539.png">

Reply:
try this:
``` js
path={{ pathname: `/requests/new/make-a-request?id=${defaultWareID}`, query: {} }}
```

Reply:
its so close... it won't load the page like this for some reason though. it is encoding the question mark to %3F, will look at if its possible to stop that from happening unless you have another suggestion 😕

<img width="787" alt="image" src="https://user-images.githubusercontent.com/73361970/215788144-2b0966db-2085-4cda-a849-5ef0f1e3d7fb.png">

Reply:
got it- it was 

`path={{ pathname: `/requests/new/make-a-request`, query: { id: defaultWareID } }}`

pushed it up to https://github.com/scientist-softserv/webstore/pull/165